docs/references/writers/index.rst

```diff
@@ -7,4 +7,3 @@ Writers
    fileio
    twig
    checkstyle
```

Comment:
checkstyle was based on xml isn't it? So this should be removed

Reply:
I thought I checked that yesterday and checkstyle is it's own thing. Can't hurt to verify during my lunch break